src/shared/sam/debugger/pythonSamDebug.ts

```diff
@@ -153,46 +150,9 @@ export async function invokePythonLambda(
 }
 
 export async function waitForPythonDebugAdapter(debugPort: number, timeout: Timeout, channelLogger: ChannelLogger) {
-    const logger = getLogger()
-
-    logger.verbose(`Testing debug adapter connection on port ${debugPort}`)
-
-    let debugServerAvailable: boolean = false
-
-    while (!debugServerAvailable) {
-        const tester = new PythonDebugAdapterHeartbeat(debugPort)
-
-        try {
-            if (await tester.connect()) {
-                if (await tester.isDebugServerUp()) {
-                    logger.verbose('Debug Adapter is available')
-                    debugServerAvailable = true
-                }
-            }
-        } catch (err) {
-            logger.verbose('Error while testing: %O', err as Error)
-        } finally {
-            await tester.disconnect()
-        }
-
-        if (!debugServerAvailable) {
-            if (timeout.remainingTime === 0) {
-                break
-            }
-
-            logger.verbose('Debug Adapter not ready, retrying...')
-            await new Promise<void>(resolve => {
-                setTimeout(resolve, PYTHON_DEBUG_ADAPTER_RETRY_DELAY_MS)
-            })
-        }
-    }
-
-    if (!debugServerAvailable) {
-        channelLogger.warn(
-            'AWS.sam.local.invoke.python.server.not.available',
-            'Unable to communicate with the Python Debug Adapter. The debugger might not succeed when attaching to your SAM Application.'
-        )
-    }
+    await new Promise<void>(resolve => {
+        setTimeout(resolve, 1000)
```

Comment:
should we change usage of `waitForPythonDebugAdapter` to `waitForDebugPort` instead? might as well re-use the logic there instead of a flat wait

Reply:
That seems to work, done.

(Tested with this PR cherry-picked onto [feature/apigateway](https://github.com/aws/aws-toolkit-vscode/commits/feature/apigateway)).

Reply:
Hmm. Using waitForDebugPort (edb8dc9) fails integration tests. Using the flat wait (536285b) passes...

Reply:
maybe the act of poking the port causes the debugger to continue/terminate before we have a chance to?